django/core/management/commands/startproject.py

```diff
@@ -19,3 +22,10 @@ def handle(self, **options):
         options["secret_key"] = SECRET_KEY_INSECURE_PREFIX + get_random_secret_key()
 
         super().handle("project", project_name, target, **options)
+
+        if target is None:
+            target = os.getcwd()
+        else:
+            target = os.path.abspath(target)
+
+        sys.stdout.write(f"Success! Created {project_name} at {target}\n")
```

Comment:
We should use `self.stdout`.
```suggestion
        self.stdout.write(f"Success! Created {project_name} at {target}.")
```